std/python/Syntax.hx

```diff
@@ -36,11 +36,11 @@ extern class Syntax {
 	static var self = macro python.Syntax;
 	#end
 
-	@:noUsing macro public static function importModule(module:String):haxe.macro.Expr {
+	@:noUsing macro static function importModule(module:String):haxe.macro.Expr {
 		return macro($self.code($v{"import " + module}) : Void);
 	}
 
-	@:noUsing macro public static function importAs(module:String, className:String):haxe.macro.Expr {
```

Comment:
Macro methods become private without `public` accessor even on extern types.

Reply:
Wat...

Reply:
Just tested and found that it works like that ¯\_(ツ)_/¯

Reply:
Is that really intentional? That might be worth talking about / changing.

Reply:
In theory you could have an `inline` function which calls a macro that is supposed to be `private`...

But by the same token, you could have an `inline` function which calls another `inline` function that is supposed to be `private`, so I'm not sure.

Reply:
Not quite sure what this has to do with inlining.

Reply:
You're allowed to have inline functions on externs, so they can have code that calls macros.

Reply:
But how is that related to the question of what the default visibility of macro functions in externs should be?

Reply:
My point is that you could _want_ a non-public `macro` (and `inline`) function in such classes, because you could have legitimate code that calls it and don't want anyone else to call it.

Reply:
But then you'd just have to add an explicit `private`, like with any other private fields in externs.

Reply:
#9209